src/scripts/main.js

```diff
@@ -1,3 +1,37 @@
 'use strict';
 
 // write your code here
+const promise1 = new Promise((resolve, reject) => {
+  const logo = document.querySelector('.logo');
+
+  logo.addEventListener('click', () => {
+    resolve('Promise was resolved!');
+  });
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(() => {
+    reject(new Error('Promise was reject'));
```

Comment:
There is a typo in the error message: 'Promise was reject' should be 'Promise was rejected'.